circular_linked_list/circular_linked_list.cpp

```diff
@@ -0,0 +1,45 @@
+#include<iostream>
+
+using namespace std;
+
+struct node{
+    int data;
+    node* next;
+};
```

Comment:
Tab found; better to use spaces  [whitespace/tab] [1]

*Origin: CPPLintBear, Section: `all.cpp`.*